checker/validate.mli

```diff
@@ -8,4 +8,6 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-val validate : bool -> Values.value -> 'a -> unit
+open Analyze
+
+val validate : debug:bool -> Values.value -> data * obj LargeArray.t -> unit
```

Comment:
nit: qualifying instead of `open` may be nicer here

Reply:
Will do.

Reply:
Ah, actually no. You have to add `Analyze` in front of `data`, `obj` *and* `LargeArray.t`, leading to a huge line. So I prefer to let it that way.